jbtcentraltarget/multiple/jbtcentral-multiple.target

```diff
@@ -70,6 +70,8 @@
       <unit id="org.aspectj.weaver" version="1.9.6.202004271746"/>
       <unit id="org.eclipse.equinox.weaving.sdk.feature.group" version="1.2.0.202004271746"/>
       <unit id="org.eclipse.equinox.weaving.hook" version="1.2.200.v20180827-1235"/>
+
```

Comment:
updated TP to add this missing bundle causing installs to fails.

Reply:
yedit is to be removed so I think we should not add it here but remove it from the studio definition

Reply:
without yedit on central, spring IDE cannot be installed, causing our jobs to fails:
https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Devstudio/view/devstudio_master/job/jbosstools-composite-install_master/lastFailedBuild/parsed_console/job/jbosstools-composite-install_master/101106/parsed_console/log_content.html#ERROR2